public/layout/default.hbs

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="/public/materialize/css/materialize.min.css">
   {{/if}}
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Henny+Penny|Montserrat+Alternates">
+  <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
```

Comment:
Why do you need axios? Right now your blocking your page load until it is loaded / parsed (1500 lines unminified).

You should consider not bringing in something like this for a small project. 

For now you should at least put the script at the bottom of the body.